cdc/changefeed.go

```diff
@@ -706,42 +726,34 @@ func (c *changeFeed) handleDDL(ctx context.Context) error {
 		c.ddlJobHistory = c.ddlJobHistory[1:]
 		c.ddlExecutedTs = todoDDLJob.BinlogInfo.FinishedTS
 		c.ddlState = model.ChangeFeedSyncDML
+		c.ddlEventCache = nil
 		return nil
 	}
 
-	executed := false
 	if !c.cyclicEnabled || c.info.Config.Cyclic.SyncDDL {
```

Comment:
c.ddlJobHistory, c.ddlExecutedTs and c.ddlState are not updated if `c.cyclicEnabled` is true and `c.info.Config.Cyclic.SyncDDL` is false

Reply:
thanks!

Reply:
I have fix these code.
Please help me to review it again.